### ISSUE TITLE
Test for WCAG success criterion 1.4.2: Audio Control

### ISSUE THREAD

**Issue by hannolans** [quailjs/quail#207]
http://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-dis-audio.html
If any audio on a Web page plays automatically for more than 3 seconds, either a mechanism is available to pause or stop the audio, or a mechanism is available to control audio volume independently from the overall system volume level. 
## Applicable

Page with
- audio element
- video element with sound
- object with sound
- embed with sound
## Not applicable

Page without video or audio content. That is in any case a page without any of the following elements:
- video
- audio
- object
- embed
## Failure
- [ ] F23 (#247): Playing a sound longer than 3 seconds where there is no mechanism to turn it off
## Techniques
- [ ] G60 (#246): Playing a sound that turns off automatically within three seconds
- [ ] G170 (#243): Providing a control near the beginning of the Web page that turns off sounds that play automatically
- [ ] G171 (#208): Playing sounds only on user request
## Relationship
- F23 is kind of counterpart of G60+G170
- If G171 is used, G60, G170 and F23 is not applicable
- First implement G171
